features/FEATURE_LWIP/lwip-interface/lwip-eth/arch/TARGET_STM/TARGET_STM32F4/TARGET_UBLOX_EVK_ODIN_W2/stm32f4_eth_conf.c

```diff
@@ -48,8 +47,7 @@ void _eth_config_mac(ETH_HandleTypeDef *heth)
         .ReceiveFlowControl = ETH_RECEIVEFLOWCONTROL_DISABLE,
         .TransmitFlowControl = ETH_TRANSMITFLOWCONTROL_DISABLE,
         .VLANTagComparison = ETH_VLANTAGCOMPARISON_16BIT,
-        .VLANTagIdentifier = 0x0U,
-        };
+        .VLANTagIdentifier = 0x0U, };
```

Comment:
@andcor02 When I commented that the terminating brace should be underneath the ETH, I meant on it's own line and directly underneath the E of ETH_MACInitTypeDef macconf . I don't think there should be a comma after the last entry either.

Reply:
See changes